### DISCUSSION TITLE
小狼毫，离完美只差一个「云取词」了吧。

### DISCUSSION THREAD

**Discussion by kwindva** [rime/home#1721] [Ideas]
之前一直用百度输入法，本身的词库很弱，但云取词太香了。
其他输入法虽然也有云取词，比如必应、搜狗、讯飞，但是在自定义双拼模式下，都不能或者不能正确云取词，只有百度输入法可以。
可惜百度输入法自从某次升级后，双拼模式下也不能云取词了，两年来的每次升级都没解决这个问题。
不知道能不能弄个功能，能调用这些大输入法供应商的云取词API。。。

**Comment by lotem** [comment]
不做。
大廠不支持自定義輸入方案。
